### ISSUE TITLE
config_path ignored, connecting to wrong cluster (Jenkins tmp credentials)

### ISSUE THREAD

**Issue by sirkubax** [hashicorp/terraform-provider-kubernetes#835]
### TL;DR
Terraform fail to load `kube/config` pointed by config_path when they injected dynamically by Jenkins into workspace@tmp directory

### Terraform Version and Provider Version

```
* provider.gitlab: version = "~> 1.0"
* provider.kubernetes: version = "~> 1.3"
* provider.null: version = "~> 1.0"
* provider.random: version = "~> 2.0"
Terraform v0.11.11
terragrunt version v0.16.3
```
### Terraform Configuration Files
```hcl

provider "kubernetes" {                                                         
  load_config_file = true                                                       
  config_path      = "${var.kubernetes_config_path}"                      
  config_context   = "${var.kubernetes_context}"                                
} 

```

The path is injected dynamically by Jenkins, and is located in @tmp workspace eg:
```
echo $TF_VAR_kubernetes_config_path 
/var/jenkins_home/jobs/admin/jobs/provision_infrastructure_for_service/workspace@tmp/secretFiles/accd81d8-2f74-46f9-aa5e-9c81492a0f9d/kube_config 
```

### Expected Behavior
Kubernetes should pick up config, and should NOT connect to localhost kubernetes

### Actual Behavior
```
* kubernetes_secret.redis_secret: Post http://localhost/api/v1/namespaces/dev-demo/secrets: dial tcp 127.0.0.1:80: connect: connection refused
* module.uat_redis.kubernetes_service.redis: 1 error(s) occurred:
```
### What actually happened?
Kubernetes failed to load kube_config, probably due to @ in the filepath (though if you create a symlink like ~/.kube/somefile or ~/.kube/config it would still fail)

Quickfix is to copy file to a known location
```
cp $kubernetes_config_path ~/.kube/config
```
